### PR TITLE
Gambit Campaigns API fixes

### LINKED ISSUE #788: Add status back into GET campaigns response
The `status` property was removed from the `Campaign` model in #777 when we made the change to query the Phoenix API anytime we want information about a Campaign.

There are Quicksilver consumers that rely on its presence:
https://github.com/DoSomething/mbc-registration-mobile/pull/65
https://github.com/DoSomething/Quicksilver-PHP/pull/104

We'll need to query the Phoenix API for each campaign the Gambit GET `campaigns` endpoint returns. This will come in handy as well when it's time to built out functionality for [Slothbot](https://github.com/dosomething/slothbot) to support letting staff know what campaigns are live and their keywords.

### PR DESCRIPTION
**Pull request by aaronschachter**
#### What's this PR do?
* Adds `status` back into the Campaign response, via querying the Phoenix to get the campaign's status
* Fixes borked `campaign/:id/message` -- missed this in #787 

#### How should this be reviewed?
Verify **GET** `v1/campaigns` returns Campaign status
Verify **POST** `v1/campaigns/:id/message` returns expected results

#### Any background context you want to provide?
Looking to use Contentful to get the copy for the relative reminder messages, instead of the `Campaign` model

#### Relevant tickets
Fixes #788

#### Checklist
- [x] Tested on staging.

### CODE REVIEW

**Review by sergiitk — APPROVED** [review]
👍

File: app/routes/campaigns.js [hunk 1]
```diff
@@ -90,37 +100,62 @@ router.post('/:id/message', (req, res) => {
...
+      mobilecommons.send_message(phone, messageBody);
+      const msg = `Sent text for ${campaignId} ${type} to ${user.mobile}`;
+      logger.info(msg);
+      stathat('Sent campaign message');
+
+      return helpers.sendResponse(res, 200, msg);
+    })
+    // TODO: Catch UnprocessibleEntityError. Commenting this code out because it is
+    // throwing a "Class constructors cannot be invoked without 'new'"" error
+    // .catch(UnprocessibleEntityError, (err) => {
```
Comment (aaronschachter) [inline app/routes/campaigns.js]: This is the weirdest thing. I'm copying https://github.com/DoSomething/gambit/blob/3ca61cfc71be5c54e871ee271cfd67b7a1dfa8b1/app/routes/chatbot.js#L304 which doesn't throw this error.

Comment (sergii-tkachenko) [inline app/routes/campaigns.js]: Might be because something returns normal Promise, the one that isn't wrapped into Bluebird.

Comment (aaronschachter) [inline app/routes/campaigns.js]: Hmm good call... potentially the `phoenix` or `northstar` Superagent clients?

Comment (sergii-tkachenko) [inline app/routes/campaigns.js]: Might be. We need to look how filtered catch actually works in Bluebird: http://bluebirdjs.com/docs/api/catch.html#filtered-catch

**Pull request merged**